core/commands/ping.go

```diff
@@ -41,7 +41,7 @@ trip latency information.
 		cmds.StringArg("peer ID", true, true, "ID of peer to be pinged.").EnableStdin(),
 	},
 	Options: []cmds.Option{
-		cmds.IntOption("count", "n", "Number of ping messages to send."),
+		cmds.IntOption("count", "n", "Number of ping messages to send.").Default(10),
```

Comment:
since we're setting the default value here, we should drop the logic in the `Run` function for manually setting the default value. The `if found {...` bits